db/migrate/20160907093023_create_matches.rb

```diff
@@ -0,0 +1,10 @@
+class CreateMatches < ActiveRecord::Migration[5.0]
+  def change
+    create_table :matches do |t|
+      t.string :game_name
+      t.string :result
```

Comment:
Are you sure you want to score results as a string ?  in a tournament how will you easily know  who won? you will have to parse the string